PREREQS.md

```diff
@@ -0,0 +1,39 @@
+## Prerequisites
+
+### AMI Acceptance
+
+Before you start, make sure the AMI that is listed in the `backendless_chef.yaml` for your region has been accepted/approved for use in the AWS Marketplace. We are using native Amazon Linux AMIs, the most up to date list is [always here](https://aws.amazon.com/amazon-linux-ami/).
+
+### For SSL
+1. You must have a DNS Record Name and Zone Name defined prior to setup if you want SSL.
```

Comment:
`defined` doesn't feel like the correct word here.     How about something like this:

1.  You must have a DNS name planned for this cluster to match the SSL certificate.
  - If you're using AWS Route 53, fill in the `Route53RecordName` and `Route53HostedZone` parameters for the template
  - Otherwise, create the DNS record in your own nameservers to be a `CNAME` record that points at the stack's Application Load Balancer (e.g. `my-chef-stack-1942464223.us-west-2.elb.amazonaws.com`)

Reply:
I'm just gonna life what you wrote :)